### DISCUSSION TITLE
Self-Sacrifice

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#47] [Card Balance]
Self-Sacrifice Dark Legion [U]
INFLUENCE 
Discard Self-Sacrifice - you may negate the effects of any just-played CONDITION, Proclamation, or Fate card, provided you first Annihilate two random cards from your hand. Any Actions or [DP] spent to play the are lost.

Someone also mentioned that this can be activated by annihiating a warrior instead of cards from hand. This can be other option to consider.

**Comment by 21-ric** [comment]
I'm really happy to see this card here. 
We can kill a match with this card. 
Especially with an imperial deck with "miscommunication", "young guard", "fill em up", "paladine's favor".
@TomekPL-1 I like your idea to shift "self-sacrifice" to DARK LEGION.
@Sol-au-fein I really like your idea : Support/Influence. Discard Heretic to negate Fate, Support or Condition
Players could play "shrieketh" for bring into play HERETIC.

**Reply by Sol-au-fein** [reply]
Spot on.

**Comment by Sol-au-fein** [comment]
Thematicaly it fits best to Heretics. Which is my favourite solution.
But @TomekPL-1 suggestion of single use support would be sufficient too.

**Comment by TomekPL-1** [comment]
I think that Sol-au-fein idea is better than my suggestion. Altgough I want to propose small changes:

Self-Sacrifice Dark Legion [U]
INFLUENCE
Once per turn you may negate the effects of any just-played CONDITION, Proclamation, or Fate card, provided you first Annihilate Dark Legion Warrior from your SQUAD. Any Actions or [DP] spent to play the are lost.

If you have 3 x self-sacrifice you can miscommunicate 3 times per turn.

**Reply by Sol-au-fein** [reply]
Single use support or restriction on heretics only would be more balanced. With your suggestion you have almost limitless uses sacrificing legionnaires. Im taking portal of undeath into consideration. Thats not very balanced compared to other factions.
Imperial has only 3 Young Guards + chance to bring some back from discard pile. Mishima has to pay 6DP for It.

**Reply by TomekPL-1** [reply]
Valid point. Could be also : annihilate NON TOKEN Dark Legion Warrior 

Single Use sounds fine to me - limiting only to heretics makes decks without them less attractive.

**Reply by 21-ric** [reply]
Once per turn sounds good to me.
You can still build a strong ALGEROTH deck without HERETIC.
And we know some DARK LEGION cards need an update to be playable.

**Comment by TomekPL-1** [comment]
From all suggestions about this card I like most this Sol-au-fein idea:

Self-Sacrifice Dark Legion [U]
CONDITION FACTION
Discard WARRIOR to which Self-Sacrifice is attached immediately after opponent plays a CONDITION, Proclamation, or Fate card, provided you first Annihilate Dark Legion Warrior from your SQUAD. Any Actions or [DP] spent to play the are lost.

**Reply by Sol-au-fein** [reply]
Ok, objections anyone?
I think, this solution is well balanced. Im only dissapointed Heretics are out of it.

**Reply by Kite80** [reply]
If it's Dark Legion faction should also include heretics

**Comment by Kite80** [comment]
Another option would be something like secret headquarters: self scarifice entering play with a single counter, you can remove a counter and annihilate 2-3 cards on hand (or a warrior in play instead, if you like best) and counter a fate card; to recharge spend 1 action (if not enough, 1 action and 3DP). And is stays for all factions.

**Comment by Sol-au-fein** [comment]
Self-Sacrifice (DARK LEGION)
ATTACHMENT CONDITION 
NOACTION if attached to HERETIC
You may discard this Warrior (outside of combat) to negate effects of any just played CONDITION, INFLUENCE or FATE card.
